File: rb/utility/prepare_headers.rb
# HealthcareGovContent SDK utility: prepare_headers
require_relative 'struct/voxgig_struct'
module HealthcareGovContentUtilities
  PrepareHeaders = ->(ctx) {
    options = ctx.client.options_map
    headers = VoxgigStruct.getprop(options, "headers")
    return {} unless headers
    out = VoxgigStruct.clone(headers)
    out.is_a?(Hash) ? out : {}
  }
end

File: .sdk/tm/php/utility/PrepareHeaders.php
<?php
declare(strict_types=1);

// HealthcareGovContent SDK utility: prepare_headers

class HealthcareGovContentPrepareHeaders
{
    public static function call(HealthcareGovContentContext $ctx): array
    {
        $options = $ctx->client->options_map();
        $headers = \Voxgig\Struct\Struct::getprop($options, 'headers');
        if (!$headers) {
            return [];
        }
        $out = \Voxgig\Struct\Struct::clone($headers);
        return is_array($out) ? $out : [];
    }
}
